recipes/taskflow/all/test_package/CMakeLists.txt

```diff
@@ -4,11 +4,13 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-set(TASKFLOW_CXX_STANDARD 14)
-if (TASKFLOW_VERSION LESS_EQUAL "2.2.0")
-    set(TASKFLOW_CXX_STANDARD 17)
+find_package(taskflow REQUIRED)
```

Comment:
```suggestion
find_package(Taskflow REQUIRED)
```

Reply:
OMG, case sensitivity... that why it don't fails on windows...